cmd/serverless-init/cloudservice/appservice.go

```diff
@@ -57,6 +58,7 @@ func (a *AppService) Init() error {
 }
 
 func isAppService() bool {
-	_, exists := os.LookupEnv(RunZip)
-	return exists
+	_, exists_linux := os.LookupEnv(RunZip)
+	_, exists_win := os.LookupEnv(AppLogsTrace)
+	return exists_linux || exists_win
```

Comment:
just a nit to use camel case for these variables

Reply:
fixed